pulp_smash/tests/rpm/api_v2/test_upload_publish.py

```diff
@@ -149,6 +150,61 @@ def _get_pkglist(update):
     return out_pkglist
 
 
+class UploadDrpmTestCase(utils.BaseAPITestCase):
+    """Test whether one can upload and associate DRPMs.
+
+    Steps:
+
+    1. Create a RPM repo.
+    2. Upload DRPM to repo.
+    3. Check if DRPM has been uploaded.
+
+
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        """Upload DRPM to a repo.
+
+        This test case targets `Pulp Smash #336`.
+        https://github.com/PulpQE/pulp-smash/issues/336
```

Comment:
Here's one way to write a link in reStructuredText:

    `my link <http://example.com>`_

Here's another way:

    `my link`_

    .. _my link: http://example.com

See [A reStructuredText Primer](http://docutils.sourceforge.net/docs/user/rst/quickstart.html).